client/dashboard/task-list/tasks/products.js

```diff
@@ -37,7 +37,7 @@ const subTasks = [
 		after: <i className="material-icons-outlined">chevron_right</i>,
 		onClick: () => recordEvent( 'tasklist_add_product', { method: 'import' } ),
 		href: getAdminLink(
-			'edit.php?post_type=product&page=product_importer&wc_onboarding_active_task=products'
+			'edit.php?post_type=product&page=product_importer&wc_onboarding_active_task=product-import'
```

Comment:
I was originally thinking we could keep the `wc_onboarding_active_task` as `products` for any product related task.  But feel free to leave this as is if you feel it offers benefit or better readability.

Reply:
The problem is that `products` will be considered complete by the time the `done` page loads, because products will exist. Using a separate key here that gets cleared when the import is done makes it much easier to hook in and update the button.